javascripts/map.js

```diff
@@ -1519,8 +1519,11 @@ function printDirections(){
 
 //determine which stylesheet to use, mobile or desktop
 function isMobile() {
-  try{ document.createEvent("TouchEvent"); return true; }
-  catch(e){ return false; }
+	if (navigator.userAgent.match(/Mobi/))
```

Comment:
Part of it is screensize, but also making sure the print option isn't shown since mobile browsers (generally) can't print.
Would probably be better to refactor and have the menus based on screen size, and detect the ability to print before showing those buttons.
Other issue is where do you draw the line between a mobile resolution and a small desktop resolution; I don't like the idea of being forced to use the mobile menus if I make my browser small, but it may be worth it in the end.